liveblog.php

```diff
@@ -332,40 +359,74 @@ public static function ajax_entries_between() {
 			self::send_user_error( __( 'A timestamp is missing. Correct URL: <permalink>/liveblog/<from>/</to>/', 'liveblog' ) );
 		}
 
+		// Get liveblog entries within the start and end boundaries
+		$result_for_json = self::get_entries_by_time( $start_timestamp, $end_timestamp );
+
+		self::json_return( $result_for_json );
+	}
+
+	/**
+	 * Get Liveblog entries between a start and end time for a post
+	 *
+	 * @param int $start_timestamp  The start time boundary
+	 * @param int $end_timestamp  	The end time boundary
+	 *
+	 * @return An array of Liveblog entries, possibly empty.
+	 */
+	public static function get_entries_by_time( $start_timestamp, $end_timestamp ) {
+
+		// Set some defaults
+		$latest_timestamp  = null;
+		$entries_for_json  = array();
+
 		// Do not cache if it's too soon
 		if ( $end_timestamp > time() ) {
-			$response_args['cache'] = false;
+			//$response_args['cache'] = false; --> Potentially Depricated
```

Comment:
Code that isn't used or needed should be removed completely, rather than commenting it out. If it's needed in the future it can be restored via the Git history.